README.md

```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
  ```bash
 sudo ./single-node.sh build ../cusdeb
 ```
-Project depends on some code which is not yet published in GitHub
+
+
+## Usage
+Script supports the following parameters:
+
+| Parameter    | Description |
+|------------|---------|
+| build target_directory | Setup all CusDeb services with all dependences to the 'target_directory'. It supposed to be empty.
```

Comment:
i think that would be a little bit more clear if we replace `build target_directory` with `build <target_directory>`